packages/twenty-server/src/engine/integrations/environment/environment-variables.ts

```diff
@@ -359,6 +362,21 @@ export class EnvironmentVariables {
 
   OPENROUTER_API_KEY: string;
 
+  LLM_CHAT_MODEL_DRIVER: LLMChatModelDriver = LLMChatModelDriver.OpenAI;
+
+  OPENAI_API_KEY: string;
+
+  LLM_PROMPT_TEMPLATE_DRIVER: LLMPromptTemplateDriver =
+    LLMPromptTemplateDriver.File;
+
+  PROMPT_TEMPLATE_NAME_ASK_AI = 'ask-ai';
```

Comment:
Not prefixed by LLM like other. Also the name doesn't really speak to me "ask-ai"? Is it ever going to be something else? What does it represent / how is it going to evolve?